packages/loot-core/src/mocks/budget.ts

```diff
@@ -563,7 +562,7 @@ export async function createTestBudget(handlers) {
   await db.runQuery('DELETE FROM categories;');
   await db.runQuery('DELETE FROM category_groups');
 
-  let accounts: AccountEntity[] = [
+  let accounts: { name: string; offBudget?: 1; id?: string }[] = [
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏do we even need a type here? Isn't it already implied from the data struct we provide it?

Reply:
Yes, because otherwise the `id` property won’t be inferred